porting/install-build/reference-rootfs.rst

```diff
@@ -50,26 +50,11 @@ The command for flashing is::
 Install rootfs and system.img
 -----------------------------
 
-Currently the latest rootfs is available at bshah's personal server: `Link <http://bshah.in/halium/halium-rootfs-20170630-151006.tar.gz>`_
+* Download the halium reference rootfs from `bshah's personal server <http://bshah.in/halium/halium-rootfs-20170630-151006.tar.gz>`_.
+* Get the halium-install script from the `JBB's repository <https://gitlab.com/JBBgameich/halium-install/>`_.
```

Comment:
I think it might make sense to promote the single-file version here instead: https://github.com/JBBgameich/halium-install/releases/download/continuous/halium-install-standalone.sh

Reply:
mhm, well, I guess this is a matter of personal taste ... with scripted language I prefer to keep users close to the source

scripted languages make it so easy to run/debug and fix/improve/contribute. to me it feels the "standalone" version just puts a barrier between debug and fix, making it harder to contribute without any real advantage (visible to me)

Reply:
I think we can leave the repo for now. People able to deal with Android´s repo command and a s...t load of Git repos should have no issues with that ^^